modules/objdetect/src/face_detect.cpp

```diff
@@ -40,6 +40,50 @@ class DNNFaceDetectorImpl : public DNNFaceDetector
         generatePriors();
     }
 
+    void setInputSize(const Size& input_size) override
+    {
+        img_w = input_size.width;
+        img_h = input_size.height;
+    }
```

Comment:
regenerate the priors ?

Reply:
Done at https://github.com/opencv/opencv/pull/20422/commits/2416377176bea00c612f677d7f0ec7b326c3cd36.